src/CTA.WebForms2Blazor/Helpers/ControlHelpers/UniversalDirectiveAttributeMap.cs

```diff
@@ -14,29 +15,34 @@ public static class UniversalDirectiveAttributeMap
         public const string CodeBehind = "CodeBehind";
         public const string ClassNameAttr = "ClassName";
         public const string LanguageAttr = "Language";
+        public const string SourceAttr = "Src";
 
         // Decided to use a universal attribute map because many of the attributes seem to be shared between
         // directives and using attribute allow lists by directive seemed like a better solution
-        public static IDictionary<string, Func<string, IEnumerable<DirectiveMigrationResult>>> AttributeMap
+        // Func takes in a string array to allow for variable number of string inputs,
+        // which may be required for different attributes
+        public static IDictionary<string, Func<string[], IEnumerable<DirectiveMigrationResult>>> AttributeMap
         {
             get
             {
                 // NOTE: Directive attributes appear to be case-insensitive
-                return new Dictionary<string, Func<string, IEnumerable<DirectiveMigrationResult>>>(StringComparer.InvariantCultureIgnoreCase)
+                return new Dictionary<string, Func<string[], IEnumerable<DirectiveMigrationResult>>>(StringComparer.InvariantCultureIgnoreCase)
                 {
-                    [MasterPageFileAttr] = attrValue =>
+                    [MasterPageFileAttr] = input =>
                     {
```

Comment:
Right now its only possible for the input to be a string array with just one variable, but this later might be expanded to take variable length inputs including null, so I'm not sure if we should handle that now or later since it depends on how the variable length inputs are implemented as well

Reply:
Probably fine as is in that case